### PR TITLE
[11.0][MIG] ao_mrp_flattened_bom_cost_xlsx

### PR DESCRIPTION
**Pull request by MiquelRForgeFlow**
Standard migration.

### CODE REVIEW

**Comment by LoisRForgeFlow** [comment]
conflicts + screenshots

**Comment by LoisRForgeFlow** [comment]
I want to do a rework to make this module depend on `mrp_flattened_bom_cost_xlsx` - **Done**

**Review by LoisRForgeFlow — APPROVED** [review]
Finally ready to go!

File: ao_mrp_flattened_bom_cost_xlsx/__manifest__.py [hunk 1]
```diff
@@ -7,9 +7,9 @@
     'version': '11.0.1.0.0',
     'category': 'Manufacturing',
     'author': "Eficent",
-    'website': 'https://github.com/Eficent',
+    'website': 'https:www,eficent.com',
```
Comment (MiquelRForgeFlow) [inline ao_mrp_flattened_bom_cost_xlsx/__manifest__.py:10]: the comma 😬

Comment (MiquelRForgeFlow) [reply ao_mrp_flattened_bom_cost_xlsx/__manifest__.py:10]: and also needs the two `//`

Comment (LoisRForgeFlow) [reply ao_mrp_flattened_bom_cost_xlsx/__manifest__.py:10]: This will be a v12 improvement :rofl: :rofl: :rofl:

**Pull request merged**